### PR TITLE
Connect travis-ci to check build status

### PR DESCRIPTION
**Pull request by HunseopJeong**
- Add '.travis.yml' to connect travis-ci
- Add badge to check build status on the README

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Comment by ws29jung** [comment]
Travis check has failed. Do we need to consider that result to merge this commit?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+language: node_js
+
+node_js:
+  - "6"
+
+sudo: enabled
+
+before_install:
+  - pip install psutil
```
Comment (Kwanghyuk-Kim) [inline .travis.yml]: Is this necessary ?
I don't remember that I did run this command before running ./launch script.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - pip install requests
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      sudo: enabled
```
Comment (Kwanghyuk-Kim) [inline .travis.yml]: duplicated declaration of "sudo: enabled" ?

File: .travis.yml (hunk 1 above)
Comment (HunseopJeong) [reply .travis.yml]: launch script didn't install pip package properly on travis-ci.
So I install packcage manually.

File: .travis.yml (hunk 2 above)
Comment (HunseopJeong) [reply .travis.yml]: Oops, duplicated.

**Comment by Kwanghyuk-Kim** [comment]
@ws29jung 
> Travis check has failed. Do we need to consider that result to merge this commit?
still have an issue on you ?

**Comment by ws29jung** [comment]
@Kwanghyuk-Kim No, not anymore.
Seem like it is working fine now.

**Pull request merged**